types/cannon/index.d.ts

```diff
@@ -845,6 +849,35 @@ declare module CANNON {
 
     }
 
+    export class Trimesh extends Shape {
+
+        vertices: number[]
+        indices: number[]
+        scale: Vec3
+
+        constructor(vertices: number[], indices: number[]);
+
+        updateTree(): void;
+        getTrianglesInAABB(aabb: AABB, result: []): [];
+        setScale(scale: Vec3): void
+        updateNormals(): void;
+        updateEdges(): void;
+        getEdgeVertex(edgeIndex: number, firstOrSecond: number, vertexStore: Vec3): void;
```

Comment:
Minor:

For the type of `firstOrSecond` we could use `0 | 1`

https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/shapes/Trimesh.js#L231

Reply:
I see no problems with that